### PR TITLE
Add changelog for release 1.2

### PR DESCRIPTION
**Pull request by gnufied**
Add changelog for 1.2 release

cc @msau42 @Jiawei0227

### CODE REVIEW

File: CHANGELOG-1.2.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Release notes for 1.2.0
```
Comment (msau42) [inline CHANGELOG-1.2.md:1]: Can we move all these into another folder similar to the other sidecars?

File: CHANGELOG-1.2.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.1.0
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Fix a bug that when CSI migration is enabled and PV is using GA topology label, resizing is not work. ([#139](https://github.com/kubernetes-csi/external-resizer/pull/139), [@Jiawei0227](https://github.com/Jiawei0227))
```
Comment (msau42) [inline CHANGELOG-1.2.md:11]: should this be kind/bug? or kind/feature "add support for csi migration for GA topology labels"

File: CHANGELOG-1.2.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+### Feature
+
+- Add a feature-gate to allow users to restore size of deleted PVCs ([#140](https://github.com/kubernetes-csi/external-resizer/pull/140), [@sunpa93](https://github.com/sunpa93))
```
Comment (msau42) [inline CHANGELOG-1.2.md:15]: which feature gate specifically?

File: CHANGELOG-1.2.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+### Bug fixes
+
+- Set the value of "migrated" field in the metrics to true or false to indicate if the call is a migration enabled feature or not ([#138](https://github.com/kubernetes-csi/external-resizer/pull/138), [@nearora-msft](https://github.com/nearora-msft))
```
Comment (msau42) [inline CHANGELOG-1.2.md:19]: Can we copy the wording that we used for the other sidecars?

File: CHANGELOG-1.2.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+- Updated runtime (Go 1.16) and dependencies ([#141](https://github.com/kubernetes-csi/external-resizer/pull/141), [@pohly](https://github.com/pohly))
```
Comment (msau42) [inline CHANGELOG-1.2.md:20]: Is this really a bug fix?

File: CHANGELOG-1.2.md (hunk 3 above)
Comment (msau42) [inline CHANGELOG-1.2.md:15]: Does this have a k8s version dependency?

Comment (gnufied) [reply CHANGELOG-1.2.md:15]: It does have a dependency on 1.21 version.

Comment (gnufied) [reply CHANGELOG-1.2.md]: added a note in readme about version dependency. Good catch. We could update the version table of external-resizer but this is alpha feature, so I am not entirely sure.

File: CHANGELOG-1.2.md (hunk 5 above)
Comment (gnufied) [reply CHANGELOG-1.2.md]: moved under `### Other (Cleanup or Flake)`

File: CHANGELOG-1.2.md (hunk 4 above)
Comment (gnufied) [reply CHANGELOG-1.2.md]: done

File: CHANGELOG-1.2.md (hunk 3 above)
Comment (gnufied) [reply CHANGELOG-1.2.md]: added

File: CHANGELOG-1.2.md (hunk 2 above)
Comment (gnufied) [reply CHANGELOG-1.2.md]: added under bug.

File: CHANGELOG-1.2.md (hunk 1 above)
Comment (gnufied) [reply CHANGELOG-1.2.md]: done

File: CHANGELOG/CHANGELOG-1.2.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+- Fix a bug that when CSI migration is enabled and PV is using GA topology label, resizing is not work. ([#139](https://github.com/kubernetes-csi/external-resizer/pull/139), [@Jiawei0227](https://github.com/Jiawei0227))
+
+### Other (Cleanup or Flake)
+
+- Updated runtime (Go 1.16) and dependencies ([#141](https://github.com/kubernetes-csi/external-resizer/pull/141), [@pohly](https://github.com/pohly))
+
+### Feature
+
+- Add a feature-gate `AnnotateFsResize` to allow users to restore size of deleted PVCs ([#140](https://github.com/kubernetes-csi/external-resizer/pull/140), [@sunpa93](https://github.com/sunpa93))
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-1.2.md:25]: Also add a note here that this requires 1.21

Comment (gnufied) [reply CHANGELOG/CHANGELOG-1.2.md:25]: done

**Pull request merged**